resources/user-guides/README.md

```diff
@@ -1,11 +1,10 @@
 # Tutorials
 
-## Implementing Experience Cloud solutions in Mobile Android and iOS Swift applications
+## Implement Experience Cloud in mobile applications
```

Comment:
Thank you for your contribution @dwright-adobe! 

Experience Cloud implementations cover only a portion of the mobile extensions, so wondering if we can rename this to cover all. Here are a few renaming suggestions: 
- Implement Adobe Experience Platform Mobile SDKs in mobile application
- Implement Adobe Edge Network and Experience Cloud solutions in mobile applications

Let me know your thoughts.

Reply:
renamed to `Implement Adobe Experience Platform Mobile SDKs in mobile applications`. @dwright-adobe @calebk1 let me know your thoughts